OverlayPlugin.Core/MemoryProcessors/Combatant/Common.cs

```diff
@@ -76,10 +89,14 @@ public enum ModelStatus : int
         // And a few others, but monsters consistently use 0, 2048, and 16384.
     }
 
-    /// <summary>A byte offset by 0x19A0 from the combatant's address that describes the combatant's aggression status.</summary>
-    // It's not exactly only about the aggression status, it's that and other things embedded into it:
-    // Sheathing the weapon adds 4 to the status.
-    // Being mid-cast adds 128 to the status.
+    /// <summary>
+    /// A byte offset by 0x1EB from the combatant's address that describes the combatant's aggression status.
+    /// The enum values below are normalized to 0-3, for consistency with previous versions.
+    /// </summary>
+    // After patch 6.3, the definition has changed:
+    // 0x10: Aggressive
+    // 0x20: In combat
+    // 0x40: Weapon unsheathed
```

Comment:
You updated the comment here, but not the actual values in the enum?

Reply:
Yes, this is for the consistency with previous versions. Maybe there are downstream users previously using the enum values, and changing this could break it. Since we are not using the entire flag but only extracting a few booleans, I kept the original definition.

Or do you think it is better to change the definition to the exact value in the memory without normalizing the data?

Reply:
The way your comment reads, the following should be true/correct I believe?

```cs
    public enum AggressionStatus : byte
    {
        // The first four values are maintained for backwards compat. Consumers should be using the flags themselves instead.
        /// <summary>Indicates a combatant that doesn't aggro on sight and is out of combat.</summary>
        [Obsolete]
        Passive = 0x00,
        /// <summary>Indicates a combatant that aggros on sight and is out of combat.</summary>
        [Obsolete]
        Aggressive = 0x10,
        /// <summary>Indicates a combatant that doesn't aggro on sight and is in combat.</summary>
        [Obsolete]
        EngagedPassive = 0x20,
        /// <summary>Indicates a combatant that aggros on sight and is in combat.</summary>
        [Obsolete]
        EngagedAggressive = 0x30,

        /// <summary> If this flag is set, the combatant is aggressive on sight</summary>
        FlagAggressive = 0x10,
        /// <summary> If this flag is set, the combatant is currently engaged in combat</summary>
        FlagEngaged = 0x20,
        /// <summary> If this flag is set, the combatant has their weapon unsheated</summary>
        FlagUnsheathed = 0x40,
    }
```

In addition to this, remove the logic to normalize the `Combatant.AggressionStatus` field, just assign the value directly (in `CombatantMemory65.cs` and `CombatantMemory70.cs`).

This will throw an `obsolete` warning for anyone using the value C#-side for some reason. It's been broken for a while now (since 6.3) as far as overlays are concerned, so they clearly haven't been using the value. They can switch to using the flags as proper flags if they're going to consume the field.